### DISCUSSION TITLE
The button gets clicked and then fails the test showing a timeout error intermittently

### DISCUSSION THREAD

**Discussion by Neha-test-dev** [getgauge/taiko#1753] [Development]
I'm trying to execute my test using Gauge and Taiko. To login into the application, the username and  password are entered in a new window which is opened during the login process. The last step is to click on Yes to complete the login process. The Yes button gets clicked and logins the user successfully. The test fails most of the times with a timeout error in clicking that element("Yes")though the button was clicked. Sometimes with the same code, the tests pass successfully. Is there any issue with clicking buttons? 
As the element to be clicked is already in focus I have tried to do the Press action as well resulting with similar experience as above. Please share your inputs if you have any solution to the problem I'm currently facing.

**Comment by NivedhaSenthil** [comment]
Taiko tries to wait for events after performing actions like click.. there might have been some network call which is taking time to complete after the click. If that does not have to be waited for you could try setting `waitForNavigation` false. 

Example: 
`await click('something', {waitForNavigation:false})`

**Reply by Neha-test-dev** [reply]
Thank you NivedhaSenthil! This worked for me.
